week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -16,20 +19,24 @@ function convertToUSD() {}
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToBRL() {}
+function convertToBRL(amount) {
+  let exchangeRate = amount * 5.7;
```

Comment:
This is great, but I have a couple of minor naming suggestions :)
My immediate thought when I read the variable name `exchangeRate` is that it's the number of BRL per GBP, i.e. 5.7 - maybe this variable could be called something like `amountInBRL` instead?

Similarly, instead of `add_commission`, perhaps that could be `brlIncludingCommission`?